packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -445,9 +445,9 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     }
 
     final String javaPath = globals.platform.isMacOS ?
-        version != null && version.major >= 2020 ?
-        globals.fs.path.join(directory, 'jre', 'Contents', 'Home') :
+        version != null && version.major < 2020 ?
```

Comment:
So is this change effectively flipping the default when `version == null` from `jre/jdk/Contents/Home` to `/jre/Contents/Home`?

Reply:
oh right, that's what the description says, lol.